### PR TITLE
Provides plugins for both builders and improves params configuration.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Related to https://github.com/maliput/maliput_documentation/issues/124

### Context
- `maliput_multilane` has two builders that are embedded together within the RoadNetworkLoader plugin. This is not ideal given that both corresponds to different kind of parameters.
- The documentation of the parameters could be improved

This PR:
 - Creates a separate RoadNetworkLoader plugin for using the OnRamp builder.
 - Improve documentation regarding the parameters that can be passed to the builders.

## Bonus

[maliput_viz] An example of dealing with both plugins:

https://user-images.githubusercontent.com/53065142/208694289-4a3e7edd-3a65-4215-bb56-c1165d4fd6b6.mp4

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

File: src/maliput_multilane/road_network_builder.cc [hunk 1]
```diff
@@ -79,6 +80,27 @@ bool IsIntersectionsNodeDefined(const YAML::Node& node) {
...
+  }
+  return config;
+}
+
+std::map<std::string, std::string> RoadNetworkConfiguration::ToStringMap() const {
+  std::map<std::string, std::string> map;
+  map[params::kYamlFile] = yaml_file;
+  map[params::kYamlDescription] = yaml_description;
+  return map;
+}
```
Comment (agalbachicar) [inline src/maliput_multilane/road_network_builder.cc:102]:
```suggestion
std::map<std::string, std::string> RoadNetworkConfiguration::ToStringMap() const {
  return {{params::kYamlFile, yaml_file}, {params::kYamlDescription, yaml_description}};
}
```

**Review by agalbachicar — CHANGES_REQUESTED** [review]
PTAL.

File: src/plugin/on_ramp_road_network.cc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+// DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#include <memory>
```
[the remark this answers is not in the document]
Comment (francocipollone) [reply src/plugin/on_ramp_road_network.cc:31]: Done.

File: include/maliput_multilane/road_network_builder.h [hunk 3]
```diff
@@ -41,6 +41,15 @@ namespace multilane {
...
 /// Contains the attributes needed for building a api::RoadNetwork.
 struct RoadNetworkConfiguration {
+  /// Creates a RoadNetworkConfiguration out of a string dictionary.
+  /// @details The keys of the map are listed at @ref road_network_configuration_builder_keys.
+  /// @param road_network_configuration A string-string map containing the configuration for the builder.
+  static RoadNetworkConfiguration FromMap(const std::map<std::string, std::string>& road_network_configuration);
+
+  /// @details The keys of the map are listed at @ref road_network_configuration_builder_keys.
+  /// @returns A string-string map containing the RoadGeometry configuration.
+  std::map<std::string, std::string> ToStringMap() const;
```
[the remark this answers is not in the document]
Comment (francocipollone) [reply include/maliput_multilane/road_network_builder.h:52]: Done.

File: include/maliput_multilane/multilane_onramp_merge.h [hunk 4]
```diff
@@ -45,24 +45,20 @@ namespace multilane {
...
-  MultilaneRoadCharacteristics(double lw, double lshoulder, double rshoulder, int lnumber)
-      : lane_width(lw), left_shoulder(lshoulder), right_shoulder(rshoulder), lane_number(lnumber) {}
-
-  const double lane_width{4.};
-  const double left_shoulder{2.};
-  const double right_shoulder{2.};
-  const int lane_number{1};
+  /// @details The keys of the map are listed at @ref on_ramp_configuration_keys.
+  /// @returns A string-string map containing the MultilaneRoadCharacteristics configuration.
+  std::map<std::string, std::string> ToStringMap() const;
```
[the remark this answers is not in the document]
Comment (francocipollone) [reply include/maliput_multilane/multilane_onramp_merge.h:58]: Done.

**Review by agalbachicar — APPROVED** [review]
LGTM

**Pull request merged**